expfmt/text_parse.go

```diff
@@ -22,10 +22,10 @@ import (
 	"strconv"
 	"strings"
 
-	dto "github.com/prometheus/client_model/go"
-
-	"github.com/golang/protobuf/proto" //nolint:staticcheck // Ignore SA1019. Need to keep deprecated package for compatibility.
 	"github.com/prometheus/common/model"
+
+	dto "github.com/prometheus/client_model/go"
+	"google.golang.org/protobuf/proto"
```

Comment:
Same nitpicking as before. This has to be

```
"google.golang.org/protobuf/proto"

dto "github.com/prometheus/client_model/go"

"github.com/prometheus/common/model"
```

And yes, it was wrong in a different way before...